releasenotes/notes/add-globalR-gates-13dd2860618e5c3f.yaml

```
@@ -0,0 +1,26 @@
+---
+features:
+  - |
+    Added global R (GR) gates, in addition to global Rx (GRx) and global Ry (GRy)
+    gates, derived from the GR gate, and global Rz (GRz) that is defined in a 
+    similar way to the GR gates. The global R gates are defined on a number of
+    qubits simultaneously, and act as a direct sum of R gates on each qubit.
+
+  - | 
+    The global R gates are defined in ``qiskit.circuit.library.generalized_gates``
+    in ``gr.py``. The global R gates will perform an approprate R gate on each qubit.
+    For example::
+
+      from qiskit import QuantumCircuit, QuantumRegister      
+      import numpy as np
+
+      num_qubits = 3
+      qr = QuantumRegister(num_qubits)
+      qc = QuantumCircuit(qr)
+
+      qc.GR(num_qubits=num_qubits, theta=np.pi/3, phi=2*np.pi/3)
```

I don't think this circuit method exists, but this should probably be 
```suggestion
      qc.compose(GR(num_qubits, theta=np.pi/3, phi=2*np.pi/3), inplace=True)
```